cubic-server/entities/Entity.hpp

```diff
@@ -90,9 +127,14 @@ class Entity : public utility::SharedFromThis<Entity> {
     virtual void teleport(const Vector3<double> &pos);
 
     // Drop an item when necessary (death of the entity, broken block, ...)
-    // The dropped item is determined by the loot tables
+    // The dropped item is determined by the loot tables // TODO that's not true but should be
```

Comment:
Please mark it at the start of the line that it is a todo, it will then appear in the docs.
Something like this: `// @todo the message goes here`